### PR TITLE
ADEN-11045 | feat: UAP embedded placeholder size always big

### PR DESCRIPTION
**Pull request by Tomasz-at-Fandom**

### CODE REVIEW

File: platforms/ucp-mobile/styles.scss [hunk 1]
```diff
@@ -41,21 +41,20 @@
 			position: sticky;
 			width: 100vw;
 			z-index: $new-uap-bfaa-z-index;
+			min-height: 211px;
```
Comment (dmnsobczak) [inline platforms/ucp-mobile/styles.scss:44]: What is this value? UAP can have different ratios and different sizes. It probably will work for most of use cases, but it's not ideal and can cause layout shift for some campaigns. Maybe let's introduce some additional padding and position UAP in the center?

File: platforms/shared/templates/helpers/uap-dom-manager.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ export class UapDomManager {
 		private reader: UapDomReader,
 	) {}
 
-	addClassToAdSlotPlaceholder(className: string): void {
-		this.manipulator.element(this.adSlot.element.parentElement).addClass(className);
+	addClassToAdSlot(className: string): void {
```
Comment (dmnsobczak) [inline platforms/shared/templates/helpers/uap-dom-manager.ts:18]: This is a breaking change and it conflict with the fix I prepared here:
https://github.com/Wikia/ad-engine/pull/1071/files#diff-e78632dbc387b1dfd9e869ee83eedecb275dd1dbde54932d78d8582c3910de0eR4
Let's talk and figure out merge conflicts :)

**Review by dmnsobczak — CHANGES_REQUESTED** [review]
Changes will conflict with changes/fixes in different ticket. This one will probably go second so let's discuss how to solve them.

File: platforms/shared/templates/helpers/uap-dom-manager.ts (hunk 2 above)
Comment (Tomasz-at-Fandom) [reply platforms/shared/templates/helpers/uap-dom-manager.ts:18]: We can have both of these functions if they are needed, but we should review their uses together.

File: platforms/ucp-mobile/styles.scss (hunk 1 above)
Comment (dmnsobczak) [reply platforms/ucp-mobile/styles.scss:45]: @sseyda What's the biggest (most square) UAP aspect ratio we're using?

**Review by dmnsobczak — APPROVED** [review]

File: platforms/ucp-mobile/styles.scss (hunk 1 above)
Comment (sseyda) [reply platforms/ucp-mobile/styles.scss:45]: @dmnsobczak as per slack, UAPs are using following sizes: 1600x400 for Desktop, 640x360 for mobile for impacts and 1600x160 and 640x213 for resolved state.

Comment (dmnsobczak) [reply platforms/ucp-mobile/styles.scss:45]: Thank you! @Tomasz-at-Fandom consider using `min-height: 213px;` here then to remove that micro content shift :)

Comment (Tomasz-at-Fandom) [reply platforms/ucp-mobile/styles.scss]: Thanks guys! I bumped it up to 213px 👍

**Pull request merged**